modules/tinymce/src/plugins/charmap/main/ts/ui/Dialog.ts

```diff
@@ -18,7 +18,11 @@ const open = (editor: Editor, charMap: CharMap[]): void => {
     },
     {
       type: 'collection',
-      name: 'results'
+      name: 'results',
+      aria: {
```

Comment:
@lnewson is pretty insistent on that we don't want to have `aria` as part of any spec. One of the options we considered when we were doing this was making the items themselves have the description on them, though that's less positioned to be able to take advantage of a common description element. I don't know what we should do to have this description tacked on. Probably have to push it to ARCOM.

Reply:
Pushing it to ARCOM is a good idea.

Reply:
As this has been said it's not something we want to do (re bridge changes) and needs to go back to our Accessibility vendor anyways for the button role thing, should this be closed for now? We can keep the branch around if necessary but it'd be nice to not have a long running PR sitting here that likely won't be merged (at least in its current state).